orc8r/cloud/helm/orc8r/charts/nms/templates/helpers/labels.tpl

```diff
@@ -33,3 +33,8 @@ app.kubernetes.io/name: {{ $application }}
 app.kubernetes.io/component: {{ $component }}
 app.kubernetes.io/instance: {{ $envAll.Release.Name }}
 {{- end -}}
+
+{{/* Generate selector labels */}}
+{{- define "magmalte-image-version-label" -}}
+image-version: {{ .Values.magmalte.image.tag}}
```

Comment:
Would it help if we add the chart version as well, since that is used to denote upgrades to Magma?

Reply:
+1